Update init-roles to match what is in AWS (#184)
ministryofjustice/analytical-platform-iam: modules/assume_role/README.md

> # Groups

Reviewer: ```suggestion
# `assume-role`
```

> This module creates roles that can be assigned to users in a desintaiton account.

Reviewer 2: typo destination

> This module creates roles that can be assigned to users in a destination account.

Reviewer: 'assigned' is so loose. And the way it is ordered might suggest the 'user' is in a destination account. I think this can be clearer. How about:
This module creates role in a destination account, that a user in the Landing account can assume into.
```suggestion
This module creates role in a destination account, that a user in the Landing account can assume into.
```

Author: Sounds good 👍

> This module creates role in a destination account, that a user in the Landing account can assume into.

Reviewer: ```suggestion
This module creates role in a destination account, and allow a list of users in the Landing account to assume into it.
```

Author: Applied with some minor fixes

It does this by:

- adding an IAM role to be assumed

> - allows attaching a optional, named policy

Reviewer: to all in the list of users

Author: I feel like that should be clear here already 🤷

> - adding an IAM user policy to allow the user assume a role in the named destination account/role

Reviewer: ```suggestion
- attaching an IAM policy that allows each user to assume the role in the named destination account
```

## Inputs

| Name                      | Description                                          | Type           | Default | Required |
| ------------------------- | ---------------------------------------------------- | -------------- | ------- | :------: |

> | `aws_iam_policy_document` | AWS IAM Policy Document to add to the group of users | `any`          | n/a     |   yes    |

Reviewer: ```suggestion
| `aws_iam_policy_document` | AWS IAM Policy Document to add to the list of users | `any`          | n/a     |   yes    |
```
